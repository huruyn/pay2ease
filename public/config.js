window.API_URL = "https://zippy-retail-api-dev.azurewebsites.net";