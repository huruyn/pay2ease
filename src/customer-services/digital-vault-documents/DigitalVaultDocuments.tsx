export default function DigitalVaultDocuments() {
  return (
    <div className="page-content">
      <h4>DIGITAL VAULT</h4>
      <h5 className="pb-2 pt-3 mt-2" style={{ borderBottom: '1px solid grey' }}>
        All files
      </h5>
      <div className="d-flex flex-column align-items-center">
        <div
          className="mt-4"
          style={{
            width: 60,
            height: 65,
            border: '1px dotted grey',
            textAlign: 'center',
            marginRight: 10,
          }}
        >
          P
        </div>
        <p className="mt-2" style={{ textAlign: 'center', maxWidth: 400 }}>
          You have no documents available for viewing. Your files will display
          when you open a new account and consent to electronic document
          delivery.
        </p>
        <p className="mt-2">
          {'{'}
          Result #5367
          {'}'}
        </p>
      </div>
    </div>
  );
}
