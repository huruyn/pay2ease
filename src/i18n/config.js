import i18n from 'i18next';
import { initReactI18next } from 'react-i18next';

i18n.use(initReactI18next).init({
  fallbackLng: 'en-US',
  lng: 'en-US',
  resources: {
    'en-CA': {
      translations: require('./locales/en-CA/translations.json')
    },
    'en-US': {
      translations: require('./locales/en-US/translations.json')
    },
    'es-US': {
      translations: require('./locales/es-US/translations.json')
    },
    'fr-CA': {
      translations: require('./locales/fr-CA/translations.json')
    },
  },
  ns: ['translations'],
  defaultNS: 'translations'
});

i18n.languages = ['en-CA', 'en-US', 'es-US', 'fr-CA'];

export default i18n;