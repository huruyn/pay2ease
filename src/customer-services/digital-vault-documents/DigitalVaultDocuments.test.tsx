import '../../i18n/config';
import DigitalVaultDocuments from './DigitalVaultDocuments';

const ReactTestRenderer = require('react-test-renderer');

describe('DigitalVaultDocuments Component', () => {
  it('matches the snapshot', () => {
    const tree = ReactTestRenderer.create(<DigitalVaultDocuments />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
