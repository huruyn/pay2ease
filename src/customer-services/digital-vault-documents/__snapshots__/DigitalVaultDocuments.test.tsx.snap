// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DigitalVaultDocuments Component matches the snapshot 1`] = `
<div
  className="page-content"
>
  <h4>
    DIGITAL VAULT
  </h4>
  <h5
    className="pb-2 pt-3 mt-2"
    style={
      Object {
        "borderBottom": "1px solid grey",
      }
    }
  >
    All files
  </h5>
  <div
    className="d-flex flex-column align-items-center"
  >
    <div
      className="mt-4"
      style={
        Object {
          "border": "1px dotted grey",
          "height": 65,
          "marginRight": 10,
          "textAlign": "center",
          "width": 60,
        }
      }
    >
      P
    </div>
    <p
      className="mt-2"
      style={
        Object {
          "maxWidth": 400,
          "textAlign": "center",
        }
      }
    >
      You have no documents available for viewing. Your files will display when you open a new account and consent to electronic document delivery.
    </p>
    <p
      className="mt-2"
    >
      {
      Result #5367
      }
    </p>
  </div>
</div>
`;
